tests/Date/ConstructTest.php

```diff
@@ -63,9 +63,13 @@ public function testCreateFromEmptyWithTestNow($class)
      */
     public function testCreateFromTimestamp($class)
     {
+        $tz = date_default_timezone_get();
+        date_default_timezone_set('Europe/London');
```

Comment:
Why change this one specifically?

Reply:
I wanted a non UTC timezone that wasn't on a different day from the test's existing assertion. This could be any European timezone, I just chose london arbitrarily.

Reply:
Without DST `Europe/London` is same as UTC isn't it?

Reply:
When using a timestamp, the date will not be modified regardless of local time zone.